src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -3242,7 +3240,7 @@ def update_upgrade_settings(self, mc: ManagedCluster) -> ManagedCluster:
             if override_until is not None:
                 try:
                     mc.upgrade_settings.override_settings.until = parse(override_until)
-                except Exception as ex:  # pylint: disable=broad-except
+                except Exception as ex:  # pylint: disable=broad-except # noqa: F841
```

Comment:
```suggestion
                except Exception:  # pylint: disable=broad-except
```

Maybe something like this

Reply:
also fine with that 👍